iidm/iidm-modification/src/test/java/com/powsybl/iidm/modification/scalable/ProportionalScalableTest.java

```diff
@@ -0,0 +1,416 @@
+/*
+ * Copyright (c) 2023. , All partners of the iTesla project (http://www.itesla-project.eu/consortium)
+ * This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
```

Comment:
Here there should also be the license id: SPDX-License-Identifier: MPL-2.0

Reply:
I think the licence header is not the one to use. The European iTesla project no longer exists (it ended in 2016).

Reply:
No idea why this one was configured, I switch it

Reply:
Should be 2023 + license ID

Reply:
modified